css/styles.css

```diff
@@ -0,0 +1,784 @@
+
+.body {
+  color: #333;
+  font-size: 13px;
+  line-height: 18px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.header {
+  width: 100%;
+}
+
+.bars {
+  width: 29px;
+  align-items: center;
+  align-self: center;
```

Comment:
Can you check https://getbootstrap.com/docs/4.0/utilities/flex/ for appropriate bootstrap classes that could be used here?

Reply:
Thanks, i will check it up